db/schema.rb

```diff
@@ -208,6 +229,20 @@
     t.integer "latest_nomis_booking_id"
   end
 
+  create_table "subscriptions", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.uuid "supplier_id", null: false
+    t.string "topic", default: "*", null: false
```

Comment:
Not sure what the purpose of this 'topic' field is. I'm finding it difficult to tell the difference between this 'topic' and the 'topic' in notifications/subscription relationship.

Reply:
@starswan the topic field here was intended as a filter, so that you could subscribe to a particular move or person, rather than all of them. However it is not in the requirements so I'll remove it for now